vega_sim/scenario/fuzzed_markets/agents.py

```diff
@@ -944,6 +1007,14 @@ def _create_latest_market(self, parent_market_id: Optional[str] = None):
             )
 
         self.vega.wait_for_total_catchup()
+
+        if self.perp_assure_enabled and mkt_config.is_perp():
```

Comment:
Can drop this if we just enable in genesis

Reply:
Done now